internal/contact/email.go

```diff
@@ -0,0 +1,43 @@
+package contact
+
+import (
+	"context"
+	"fmt"
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+)
+
+const separator = "### app version:"
```

Comment:
```suggestion
const separator = "\n### app version:"
```
I'd love for a new line to be inserted here. Opinion?

Reply:
In the original version there is no new line, but I'm ok with new line.
https://github.com/CollActionteam/collaction_backend/blob/7aee4ea77a93eb9c7f105d40bfd32c4d885ada33/email_contact/main.go#L121-L123

Reply:
Haha yeah, I know, that's exactly why I made the comment haha 👍